test/model_tests/user_test.dart

```diff
@@ -0,0 +1,162 @@
+import 'package:beacon/models/beacon/beacon.dart';
+import 'package:beacon/models/user/user_info.dart';
+import 'package:test/test.dart';
+
+void main() {
+  //structered according to fetchBeaconDetail Query.
+  Map<String, dynamic> dummyJson = {
+    "_id": "61fd509bf0c4c3219ce356ed",
+    "name": "asdasd",
+    "email": "asdf@g.com",
```

Comment:
Instead of adding random alphabets, I think it would be better to be more generic, like change it to "test_name"
And also since you've written the model tests it would be better to add validation check too. For ex: _id can't be null (similarly for other fields in user and beacon model)

Reply:
Sure, will do the needful!